rust/bridge/node/futures/src/promise.rs

```diff
@@ -0,0 +1,133 @@
+//
+// Copyright 2020-2021 Signal Messenger, LLC.
+// SPDX-License-Identifier: AGPL-3.0-only
+//
+
+use futures::FutureExt;
+use neon::prelude::*;
+use std::future::Future;
+use std::panic::{catch_unwind, AssertUnwindSafe, UnwindSafe};
+
+use crate::executor::ContextEx;
+use crate::util::describe_any;
+use crate::*;
+
+const RESOLVE_SLOT: &str = "_resolve";
```

Comment:
Should be FULFILL_SLOT = "_fulfill"

Reply:
This one turned out to be correct: it's the `resolve` from [the Promise constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/Promise) rather than the "fulfill" from [`then`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then).

Reply:
That looks like a spec naming failure.  But yeah, I guess stick with the spec naming.